blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMultiRollupsQueryHandlerIntegrationTest.java

```diff
@@ -0,0 +1,115 @@
+
+        // assert response content
+        String responseContent = EntityUtils.toString(query_response.getEntity(), "UTF-8");
+        String expectedResponse = "{\n" +
+                "  \"metrics\": [\n" +
+                "    {\n" +
+                "      \"unit\": \"unknown\",\n" +
+                "      \"metric\": \"3333333.G1s\",\n" +
+                "      \"data\": [\n" +
+                "        {\n" +
+                "          \"timestamp\": 1389211230\n" +
+                "        }\n" +
+                "      ],\n" +
+                "      \"type\": \"number\"\n" +
+                "    },\n" +
+                "    {\n" +
+                "      \"unit\": \"unknown\",\n" +
+                "      \"metric\": \"3333333.G10s\",\n" +
+                "      \"data\": [\n" +
+                "        {\n" +
+                "          \"timestamp\": 1389211230\n" +
+                "        }\n" +
+                "      ],\n" +
+                "      \"type\": \"number\"\n" +
+                "    }\n" +
+                "  ]\n" +
+                "}";
+
+        Assert.assertEquals(expectedResponse, responseContent);
+        EntityUtils.consume(query_response.getEntity());
+    }
+
+    @Test
+    public void testMultiplotQueryWithEnum() throws Exception {
+        // ingest and rollup metrics with enum values and verify CF points and elastic search indexes
+        final String tenant_id = "333333";
+
+        // post multi metrics for ingestion and verify
+        HttpResponse response = postAggregatedMetric(tenant_id, "src/test/resources/sample_enums_payload.json");
+        Assert.assertEquals("Should get status 200 from ingestion server for POST", 200, response.getStatusLine().getStatusCode());
+        EntityUtils.consume(response.getEntity());
+
+        // query for multiplot metric and assert results
+        HttpResponse query_response = queryMultiplot(tenant_id, fromTime, toTime, "", "FULL", "enum_values", "['enum_metric_test']");
+        Assert.assertEquals("Should get status 200 from query server for multiplot POST", 200, query_response.getStatusLine().getStatusCode());
+
+        // assert response content
+        String responseContent = EntityUtils.toString(query_response.getEntity(), "UTF-8");
+        String expectedResponse = "{\n" +
+                "  \"metrics\": [\n" +
+                "    {\n" +
+                "      \"unit\": \"unknown\",\n" +
+                "      \"metric\": \"enum_metric_test\",\n" +
+                "      \"data\": [\n" +
+                "        {\n" +
+                "          \"timestamp\": 1389211230,\n" +
+                "          \"enum_values\": {\n" +
+                "            \"v3\": 1\n" +
+                "          }\n" +
+                "        }\n" +
+                "      ],\n" +
+                "      \"type\": \"number\"\n" +
```

Comment:
The "type" here should be "ENUM" (see result in testSingleplotQueryWithEnum()).  Not sure why it's currently returning "type": "number".